spec/features/user_creates_new_book_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+feature 'User creates a new book' do
+  scenario 'successfully' do
+    visit root_path
+
+    create_book("The Lord of the Rings: The Fellowship of the Ring", "J. R. R. Tolkien" )
```

Comment:
Since this repo is public it is considered open source, so you should be able to sign up and use it for free